sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py

```diff
@@ -529,13 +530,14 @@ def create_form_client_and_container_sas_url(self, **kwargs):
 @pytest.fixture(scope="session")
 def form_recognizer_account():
     test_case = AzureTestCase("__init__")
-    rg_preparer = ResourceGroupPreparer(random_name_enabled=True, name_prefix='pycog', location="westus")
+    rg_preparer = ResourceGroupPreparer(random_name_enabled=True, name_prefix='pycog', location="westcentralus")
     form_recognizer_preparer = CognitiveServicesAccountPreparer(
         random_name_enabled=True,
         kind="formrecognizer",
         name_prefix='pycog',
-        location="westus"
+        location="westcentralus"
     )
+    time.sleep(60)  # current ask until race condition bug fixed
```

Comment:
might be worth it to only do time.sleep for live tests, and to have the cog service account preparer take in a parameter for whether to sleep after creation so not all calls to it have the time.sleep?